src/lightkurve/targetpixelfile.py

```diff
@@ -1150,6 +1151,16 @@ def plot(
                 self.row - 0.5,
                 self.row + self.shape[1] - 0.5,
             )
+            
+            #If an axes is passed that used WCS projection, don't use img_extent
+            #This addresses lk issue #1095, where the tpf was incorrectly plotted
+            try: 
+            	if hasattr(ax, "wcs"):
+            		img_extent = None
```

Comment:
I think there is a double tab here

Reply:
I don't know why it looks like that here. I checked and it is a single tab.